Test/BSN.Commons.Orm.Redis.Tests/RepositoryTest.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BSN.Commons.Orm.Redis.Tests.Mock;
+using BSN.Commons.Orm.Redis.Tests.Dto;
+using BSN.Commons.Infrastructure;
+using BSN.Commons.Infrastructure.Redis;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+
+namespace BSN.Commons.Orm.Redis.Tests
+{
+    [TestFixture]
+    public class RepositoryTest
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            _databaseFactory = CreateDatabaseFactory();
+            _userRepository = CreateUserRepository(_databaseFactory);
+        }
+
+        [Test]
+        public void AddUserToDataBaseAndNullQueue_CorrectInput_UsereShouldBeCorectlyAddedToDatabase()
```

Comment:
Where is another tests?

Reply:
other tests are using unit of work that is not yet supported here.

we can implement simillar tests without UnitOfWork it its necessary